rebar.config.script

```diff
@@ -191,7 +191,7 @@ ErlOpts = case os:getenv("ERL_OPTS") of
 end.
 
 AddConfig = [
-    {require_otp_vsn, "20|21|22|23"},
+    {require_otp_vsn, "21|22|23"},
```

Comment:
can't review the rest but wanted to explicitly say "+1 for dropping 20.x on main, -1 for dropping 20.x on 3.x." 

That means this wouldn't easily be backported to 3.x, until such time as we can confirm >20.x on 3.x code is stable at scale.